### PR TITLE
Add Vale rule for using gerunds in headings

### PR DESCRIPTION
**Pull request by jdbaldry**
Let me know if you can think of any edge cases. At the moment I'm matching for the first word ending "ing" on any line that begins with one or more `#` characters.

Ran on the `next` versions of documentation on the website and I got 915 hits.

### CODE REVIEW

File: vale/Grafana/BareInfinitives.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+extends: script
+message: |
+  For a task-based heading, start with a [bare infinitive](https://en.wikipedia.org/wiki/Infinitive#English), also known as a plain form or [base form](https://en.wikipedia.org/wiki/English_verbs#Base_form) verb.
+  In English, the imperative mood also uses the base form verb, so it looks the same as the bare infinitive.
+
+
```
Comment (clayton-cornell) [inline vale/Grafana/BareInfinitives.yml:6]: Extra CR/LF?

**Review by clayton-cornell — APPROVED** [review]

**Comment by jdbaldry** [comment]
215 unique bare infinitives. I think there may be some false positives in headings like "Billing dashboard" but they should be few and far between.

```
Accessing
Activating
Adding
Addressing
Aggregating
Alerting
Attaching
Auditing
Authenticating
Automating
Autoscaling
Backporting
Batching
Benchmarking
Billing
Blocking
Breaking
Bring
Bringing
Building
Bundling
Caching
Chaining
Changing
Checking
Choosing
Clustering
Collecting
Combining
Comparing
Configuring
Conflicting
Connecting
Constructing
Contacting
Contributing
Controlling
Converting
Counting
Creating
Dealing
Debugging
Declaring
Deduplicating
Deleting
Deploying
Deriving
Determining
Diagnosing
Disabling
Dropping
Enabling
Encountering
Encrypting
Enforcing
Enriching
Escaping
Estimating
Examining
Excluding
Existing
Expanding
Exploring
Exporting
Exposing
Extracting
Falling
Fetching
Filtering
Finding
Finishing
Fixing
Flushing
Forwarding
Generating
Getting
Granting
Grouping
Handling
Heading
Ignoring
Importing
Including
Incoming
Increasing
Indexing
Inferring
Ingesting
Initiating
Injecting
Inspecting
Installing
Interacting
Introducing
Invoking
Labeling
Launching
Learning
Licensing
Limiting
Linking
Loading
Logging
Looking
Making
Managing
Matching
Measuring
Merging
Migrating
Mirroring
Missing
Mixing
Monitoring
Moving
Naming
Navigating
Observing
Opening
Ordering
Outgoing
Overriding
Parsing
Passing
Pending
Ping
Planning
Polling
Preparing
Printing
Processing
Profiling
Providing
Provisioning
Proxying
Pushing
Putting
Querying
Queueing
Ramping
Reading
Receiving
Reconstructing
Recording
Recovering
Referencing
Releasing
Reloading
Removing
Renaming
Rendering
Repeating
Reporting
Resetting
Resizing
Restoring
Retrieving
Returning
Ring
Routing
Running
Sampling
Saving
Scaling
Scheduling
Scraping
Scripting
Scrolling
Searching
Securing
Selecting
Sending
Serving
Setting
Sharding
Sharing
Shipping
Signing
Something
Specifying
Spring
Streaming
String
Substring
Tagging
Templating
Testing
Timing
Tracing
Training
Transforming
Triggering
Troubleshooting
Understanding
Upcoming
Updating
Upgrading
Uploading
Using
Validating
Verifying
Viewing
Visualizing
Waiting
Working
Writing
```

**Comment by clayton-cornell** [comment]
Just thinking here... are these actually bare infinitives? Bare infinitives and gerunds aren't the same thing :-)  I just noticed that the Vale rule is named bare infinitives... the pedantic numpty that I am says.. nooooo... we should rename this rule to reflect that it's about gerunds not bare infinitives :-)

**Comment by jdbaldry** [comment]
Looking at the rules names, I'm not sure if we have a consistent approach to naming.

I'm happy now that I've had gerunds explained to me that we can rename this to Gerunds (it was what I called it initially and then changed haha)

**Pull request merged**